### PR TITLE
Make drag-and-drop widget's draggable state more noticeable

### PR DESCRIPTION
**Pull request by hbillings**
Addresses #397. Both the text and color will change, though TBH I'm not sure I love the color change. Feedback very welcome! 
screen shot 2016-08-05 at 5 45 39 pm

### CODE REVIEW

File: frontend/source/sass/components/_upload.scss [hunk 1]
```diff
@@ -36,7 +40,16 @@
...
   &.dragged-over {
-    background-color: $color-white;
+    background-color: $color-green-lightest;
+    border: 2px dashed $color-green;
+    .upload-chooser {
+      * {
+        visibility: hidden;
+      }
+      &:before {
+        content: "Drop files here!"
```
Comment (toolness) [inline frontend/source/sass/components/_upload.scss]: Should this be "Drop file here", since they can only upload one file at a time?

**Comment by toolness** [comment]
This looks great!  What do you think of changing the "Drop files here" text to "Drop file here" or "Drop your file here", since they can only upload one at a time?

Also, can you update the styleguide example to put a `<span>` around the "XLS, XLSX, or CSV format, please" like you did in the step 1 form?  Otherwise the styleguide example looks a bit broken when hovering over...

**Comment by hbillings** [comment]
Good catch on the language! Will update that and the styleguide.

**Comment by hbillings** [comment]
Okay, this should be ready. Plus, I added some more styling for the gracefully degraded version. 
screen shot 2016-08-08 at 10 54 37 am

**Comment by toolness** [comment]
Oooo nice, thanks for the degraded styling!  Looks much better now.

**Pull request merged**